server/lib/routes.js

```diff
@@ -46,6 +46,11 @@ module.exports = function (config, i18n) {
       res.redirect(req.originalUrl.slice(3));
     });
 
+    // handle account unlock links
+    app.get('/v1/complete_unlock_account', function (req, res) {
+      res.redirect(req.originalUrl.slice(3));
```

Comment:
To a new reader, this is terrifingly obtuse.  What's happening here and what's the significance of `slice(3)`?  My guess is that we're stripping the `/v1` prefix but that's not obvious.